.github/workflows/test-application.yaml

```diff
@@ -0,0 +1,85 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - master
+            - develop
+            - '[0-9]+.[0-9]+'
```

Comment:
Add `- '[0-9]+.x'` to this list to allow branches like `2.x`

Reply:
👍

Reply:
i just removed the `master` and `develop` branch, so we can remove them here too